content/docs/command-reference/import-url.md

```diff
@@ -10,8 +10,8 @@ import `.dvc` file is created).
 ## Synopsis
 
 ```usage
-usage: dvc import-url [-h] [-q | -v] [--file <filename>] [--no-exec]
-                      [--desc <text>]
+usage: dvc import-url [-h] [-q | -v] [-j <number>] [--file <filename>]
```

Comment:
Added to `import-url` too per https://github.com/iterative/dvc/issues/5267#issuecomment-771757774

Reply:
Wouldn't this conflict with #2091?

Reply:
If the changes are different, yes. Hard to avoid duplicity sometimes, we can solve the conflicts 🙂